tests/UnitTests/WebConfigTransformHandlerTests.cs

```diff
@@ -11,17 +11,17 @@ namespace UnitTests
     public class WebConfigTransformHandlerTests
     {
         private readonly IConfigurationRoot _configMock;
-        private readonly Mock<IWebConfigReader> _webConfigReaderMock;
-        private readonly Mock<IWebConfigWriter> _webConfigWriterMock;
-        private readonly WebConfigTransformHandler _transformHandler; 
+        private readonly Mock<IConfigReader> _webConfigReaderMock;
+        private readonly Mock<IConfigWriter> _webConfigWriterMock;
+        private readonly ConfigTransformHandler _transformHandler;
 
         public WebConfigTransformHandlerTests()
         {
             _configMock = GetMockConfiguration();
-            _webConfigReaderMock = new Mock<IWebConfigReader>();
-            _webConfigWriterMock = new Mock<IWebConfigWriter>();
```

Comment:
Much like my earlier comment. If these `class` are not longer referenced then let's delete them.

Reply:
same as earlier comment.